ui/lobby/css/_forum.scss

```diff
@@ -6,6 +6,10 @@
     white-space: nowrap;
   }
 
+  &__content {
+    max-height: 248px;
```

Comment:
```suggestion
    max-height: 16rem;
```

Reply:
yes, thanks!

Reply:
oh shoot I could have just clicked commit suggestion.
hahah I submitted it as well.